lib/rubocop/cop/style/safe_navigation.rb

```diff
@@ -151,7 +151,12 @@ def extract_parts_from_if(node)
 
           checked_variable, matching_receiver, method =
             extract_common_parts(receiver, variable)
-          matching_receiver = nil if LOGIC_JUMP_KEYWORDS.include?(receiver.type)
+
```

Comment:
I would go first by renaming `modifier_if_safe_navigation_candidate?` to just `modifier_if_safe_navigation_candidate` as it doesn't return a boolean value. Next, if the receiver is returned, it is a Node for sure, so no need for the second check (receiver.type). So it may become `matching_receiver = nil if receiver.nil? || LOGIC_JUMP_KEYWORDS.include?(receiver.type)`. Which isLOGIC_JUMP_KEYWORDS a bit long, so it may be extracted in a method

Reply:
Alright, thanks! Made those changes, just not sure about the method extracted being called `safeguarded?`, maybe not the best of names.